packages/sdk/src/utils/misc.ts

```diff
@@ -75,3 +75,26 @@ export function debounce<F extends Procedure>(
     }
   };
 }
+
```

Comment:
create a new file for this util called `src/reaction/reaction.ts` with a static method `hasBeenModified` in the `Reaction` class